sensor/CSVLoader.py

```diff
@@ -0,0 +1,8 @@
+import csv
+
+
+def generator_from_csv(file_path):
+    with open(file_path, 'r', newline='') as csvfile:
+        csv_reader = csv.DictReader(csvfile)
+        for row in csv_reader:
+            yield row
```

Comment:
Does this function need to be in a separate file?

Reply:
I would say yes, the sender file, is already overflown with functions & functionality.
For readability purposes, I think it's better to keep it on a separate file.